packages/core/src/RenderingEngine/Viewport.ts

```diff
@@ -682,6 +686,99 @@ class Viewport implements IViewport {
     return true;
   }
 
+  /**
+   * Resets the stored initial offset values for
+   * zoom and pan.  These values are the offset
+   * values used to allow the getPan and getZoom
+   * to return [0,0] and 1 for the initial values.
+   * @param resetOffsets can be passed to skip resetting, ie a no-op on this call
+   */
+  protected resetInitialOffsets(resetOffsets = true) {
```

Comment:
The whole notion of `resetInitialOffset` does not make sense to me, this function just get the camera and set it as initialCamera. Shouldn't this function be like `setInitialCamera`? and then `resetOffsets` be like `storeInitialCamera` or something else? I don't understand what is being `reset` here?

Reply:
The idea is that it stores the initial camera, so I'm going to call it just that.  Will push an update momentarily.